src/data/logbook.ts

```diff
@@ -415,6 +415,10 @@ export const localizeStateMessage = (
     return localize(`${LOGBOOK_LOCALIZE_PATH}.turned_off`);
   }
 
+  if (state === UNKNOWN) {
+    return localize(`${LOGBOOK_LOCALIZE_PATH}.became_unknown`);
+  }
+
   if (UNAVAILABLE_STATES.includes(state)) {
```

Comment:
```suggestion
  if (state === UNAVAILABLE) {
```
We can simplify this condition.